fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -55,19 +56,29 @@
     public static final String BASE_VALID_URL = "/$export";
     public static final String BASE_VALID_STATUS_URL = "/$bulkdata-status";
     public static final String FORMAT = "application/fhir+ndjson";
+    private final String tenantName = "default";
+    private final String dataStoreId = "default";
```

Comment:
Why do you have to specify the tenantName/DatastoreId?

Reply:
this is used for my local tests to make sure the per tenant configures for bulkdata working correctly.  example, I used "default" + "default", "postgresql" + "default" etc in my tests.